src/draw/lv_draw_vector_private.h

```diff
@@ -70,30 +77,53 @@ struct _lv_vector_stroke_dsc_t {
     lv_matrix_t matrix;
 };
 
-struct _lv_vector_draw_dsc_t {
+/**
+ * Stores how to fill, stroke, transform etc a given path
+ */
+struct _lv_vector_path_attr_t {
```

Comment:
Although there will be no problem with this change, `vector_draw_dsc` actually represents the "context" of this drawing. It is not a set of path attributes. A possible optimization is that paths drawn in the current context should share the same context until the attributes are changed.